example/lib/hms_app_settings.dart

```diff
@@ -460,7 +464,7 @@ class _HMSAppSettingsState extends State<HMSAppSettings> {
                             fontWeight: FontWeight.w400),
                       ),
                       trailing: Text(
-                        HMSSDKConstants.hmsSDKVersion,
+                        versions["flutter"]??"",
```

Comment:
If the version could not be found, there should be an Error log. All SDK versions should always have the correct versions defined.